Add a database TLS option to trust system CAs (#43286)
gravitational/teleport: docs/pages/includes/database-access/self-hosted-config-start.mdx

      --labels=env=dev
   ```

> If your database servers use certificates that are publicly signed by an SSL
> certificate service such as Comodo or Digicert, you can use the

Reviewer: nit

```suggestion
If your database servers use certificates that are signed by a public CA
such as ComodoCA or DigiCert, you can use the
```
